.mergify.yml

```diff
@@ -18,15 +24,6 @@ pull_request_rules:
           {{ title }} (#{{ number }})
           {{ body }}
 
-  - name: Backport patches to the release/v7.0.x branch
-    conditions:
-      - base=main
-      - label=A:backport/v7.0.x
-    actions:
-      backport:
-        branches:
-          - release/v7.0.x
-
   - name: Backport patches to the release/v8.0.x branch
```

Comment:
How about the following condition for all backports?
```
    conditions:
      - "#approved-reviews-by>=1"
```

Reply:
It's already part of the branch protection rule. That will overwrite whatever you add here.